backend/src/test/java/com/example/backend/controller/BarkdateControllerTest.java

```diff
@@ -27,4 +28,28 @@ void getAllLocations_whenCalledInitially_thenReturnEmptyList() throws Exception
                 .andExpect(status().isOk())
                 .andExpect(content().json("[]"));
     }
+
+    @Test
+    void addLocation_whenNewRestaurantIsAdded_thenReturnRestaurant() throws Exception {
+        //GIVEN
+        //WHEN&THEN
+        mvc.perform(post("/api/bark-dates")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("""
+                            {
+                                "city": "Munich",
+                                "venue": "Englischer Garten",
+                                "googlePlusCode": "123456"
+                            }
+                        """))
+                .andExpect(status().isOk())
+                .andExpect(content().json("""
+                            {
+                                "city": "Munich",
+                                "venue": "Englischer Garten",
+                                "googlePlusCode": "123456"
+                            }
+                        """))
```

Comment:
if you want to get rid of this, you can use the ObjectMapper :-) not neccessary! just a quick tip ^-^